### ISSUE TITLE
App Crashing

### ISSUE THREAD

**Issue by dustin-kl** [mihai8804858/swift-chunked-audio-player#5]
My app occasionally crashes when using the audio player. 

It throws an "EXC_BAD_ACCESS" in the following function.

```
func parseData(_ data: Data) {
        guard let audioStreamID = audioStreamID else {
            print("audioStreamID is nil")
            return
        }
        
        guard !data.isEmpty else {
            print("Data is empty")
            return
        }
        
        data.withUnsafeBytes { pointer in
            guard let baseAddress = pointer.baseAddress else { return }
            AudioFileStreamParseBytes(audioStreamID, UInt32(data.count), baseAddress, [])
        }
    }
```

More specifically it happens in this line

`AudioFileStreamParseBytes(audioStreamID, UInt32(data.count), baseAddress, [])`

**Comment by mihai8804858** [comment]
Fixed in [1.0.2](https://github.com/mihai8804858/swift-chunked-audio-player/releases/tag/1.0.2)

**Comment by dustin-kl** [comment]
Amazing!
